source/extensions/filters/listener/http_inspector/http_inspector.cc

```diff
@@ -99,6 +99,7 @@ ParseState Filter::parseHttpHeader(absl::string_view data) {
         // Set other HTTP protocols to HTTP/1.0
         protocol_ = Http::Headers::get().ProtocolStrings.Http10String;
       }
+      ENVOY_LOG(trace, "http inspector: set application protocol to %s", protocol_);
```

Comment:
So this one can be removed, right? since you already have one later.

Reply:
Sure.